python/plugins/processing/algs/qgis/Rasterize.py

```diff
@@ -57,6 +57,7 @@
 
 
 class RasterizeAlgorithm(QgisAlgorithm):
+
```

Comment:
It's PEP 257, for class docstring, they put one empty line before and after the docstring.
For function docstring, no new line before and after it.